src/ImageSharp/Formats/Bmp/BmpDecoderCore.cs

```diff
@@ -246,19 +247,21 @@ private void ReadRle8<TPixel>(PixelAccessor<TPixel> pixels, byte[] colors, int w
             where TPixel : struct, IPixel<TPixel>
         {
             var color = default(TPixel);
-            var rgba = default(Rgba32);
+            var rgba = new Rgba32(0, 0, 0, 255);
 
-            using (var buffer = Buffer<byte>.CreateClean(width * height))
+            using (var buffer = Buffer2D<byte>.CreateClean(width, height))
             {
                 this.UncompressRle8(width, buffer);
 
                 for (int y = 0; y < height; y++)
                 {
                     int newY = Invert(y, height, inverted);
```

Comment:
I think it's possible to go without the `Invert()` method by managing `newY` as a loop variable, but the benefits are not significant ... just talking my idea.